gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -0,0 +1,142 @@
+              into rows by replacing this style with <code className="nx-code">.nx-card-container--row</code> above.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Basic card element.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__header</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              Wraps the card call out (if any). The call out typically consists of a number, sparkline, icon, or
+              other small graphic. It should not be free form text.
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__call-out-text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">
+              In the case where the call out consists of text (e.g. 99%) it should be wrapped in this class (typically
+              in a <code className="nx-code">&lt;span&gt;</code>).
+            </td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__text</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Free-form text. Should be concise.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__footer</code></td>
+            <td className="nx-cell">Element</td>
+            <td className="nx-cell">Card footer. Typically provides a link to more information.</td>
+          </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-card__content</code></td>
+            <td className="nx-cell">Wrapping element</td>
+            <td className="nx-cell">
+              <code className="nx-code">.nx-card__content</code> applied to a
+              {' '}<code className="nx-code">&lt;div&gt;</code> wraps
+              {' '}<code className="nx-code">.nx-card__call-out</code> and
+              {' '}<code className="nx-code">.nx-card__call-out-text</code> in the alternate row card layout.
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Vertical Card"
```

Comment:
The names and ids here and below still reference the vertical/horizontal terms

Reply:
> The names and ids here and below still reference the vertical/horizontal terms

https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/38ec97d38d3854439f9f6e4e7fd34936d4cd01f6